docker/dockerfiles/allin1/README.md

```diff
@@ -0,0 +1,24 @@
+This docker file builds an ubuntu based all-in-one image that automatically starts then a complete Starrocks cluster stack(FE, BE and register BE to FE) at the time when container starts.
+
+This is mainly used for developer to test starrocks locally.
+Please override the [be.conf](be.conf) or [fe.conf](fe.conf) or fe.conf as you need.
+
+### Build the all in one docker image:
+
+**1) Use artifact image to package allin1 runtime container image**
+```
+DOCKER_BUILDKIT=1 docker build --build-arg ARTIFACT_SOURCE=image --build-arg ARTIFACTIMAGE=<image> -f allin1-ubuntu.Dockerfile -t allin1-ubuntu:<tag> .
+```
+**E.g.**
+```shell
+DOCKER_BUILDKIT=1 docker build --build-arg ARTIFACT_SOURCE=image --build-arg ARTIFACTIMAGE=ghcr.io/dengliu/starrocks/artifact-ubuntu:latest -f allin1-ubuntu.Dockerfile -t allin1-ubuntu:latest .
```

Comment:
```suggestion
DOCKER_BUILDKIT=1 docker build --build-arg ARTIFACT_SOURCE=image --build-arg ARTIFACTIMAGE=ghcr.io/dengliu/starrocks/artifact-ubuntu:latest -f allin1-ubuntu.Dockerfile -t allin1-ubuntu:latest .
```

It will be better if you can change `dengliu/starrocks` to `starrocks/starrocks`

Reply:
can you publish some images to ghcr via the docker file I sent you before in https://github.com/StarRocks/starrocks/pull/17890 ? 
So this script can use the image from `ghcr.io/StarRocks/`